h2o-algos/src/test/java/hex/ensemble/StackedEnsembleTest.java

```diff
@@ -1406,4 +1408,31 @@ private void familyAndLinkInferenceTestHelper(
       }
     }
   }
+
+  @Test
+  public void testStackedEnsembleDoesntIgnoreColumnIfAnyBaseModelUsesIt() {
+    try {
+        Scope.enter();
+        StackedEnsembleModel.StackedEnsembleOutput output = null;
+        try {
+            output = basicEnsemble("./smalldata/junit/cars.csv",
+                    null,
+                    new StackedEnsembleTest.PrepData() {
+                        int prep(Frame fr) {
+                            fr.remove("name").remove();
+                            Vec acVec = fr.anyVec().makeCon(Math.PI); // this is what use to break SE (SE ignored this column, DRF did not)
+                            Scope.track(acVec);
+                            acVec.setNA(0);
+                            fr.insertVec(0, "almost_constant", acVec);
+                            return ~fr.find("economy (mpg)");
```

Comment:
can we get rid of those confusing-hair-pulling-wtf-is-it-useful-here bitwise operator?
I tried to run `StackedEnsembleTest` after removing those as well as the make-things-even-more-confusing-and-useless statement:
```java
            if( idx < 0 ) idx = ~idx;
```
and all tests are passing...